shell/src/peer_manager.rs

```diff
@@ -405,6 +413,7 @@ impl
             discovery_last: None,
             check_peer_count_last: None,
             shutting_down: false,
+            pow_target,
```

Comment:
@akoptelov 
I would add pow_target to the local_node_info: 
```
local_node_info: Arc::new(LocalPeerInfo::new(
                p2p_config.listener_port,
                identity,
                shell_compatibility_version,
            )),
```

this struct is passed already to bootstrap fn, so we dont need to add extra parameter